releasenotes-builder/src/test/java/com/github/checkstyle/templates/TemplateProcessorTest.java

```diff
@@ -313,13 +313,26 @@ private static ReleaseNotesMessage createReleaseNotesMessage(int number, String
         titleField.setAccessible(true);
         titleField.set(issue, title);
 
+        final Field escapedField = issue.getClass().getDeclaredField("title");
+        escapedField.setAccessible(true);
+        escapedField.set(issue, title);
```

Comment:
isn't it the same as `titleField` defined above?

Reply:
yes, the only difference is, that `titleField` is general and escapedField will be a field that escapes the `@` if found in the GitHub Page Notes.

Reply:
I mean you update the same field twice with the same value:
```java
        final Field titleField = issue.getClass().getDeclaredField("title");
        titleField.setAccessible(true);
        titleField.set(issue, title);

        final Field escapedField = issue.getClass().getDeclaredField("title");
        escapedField.setAccessible(true);
        escapedField.set(issue, title);
```

Reply:
I am not sure if this is what @romani wants me to do. Please confirm that do I need to create an another value or title is good.

Reply:
Yes, you need to create new template field, that is used only in GitHub template.